src/Utils.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var Random = require('random-js');
+var Matrix = require('ml-matrix');
+
+
+function isInt(n) {
+    return Number(n) === n && n % 1 === 0;
+}
+
+function isString(s) {
+    return typeof s === 'string' || s instanceof String;
```

Comment:
I don't think we should support instances of the String constructor. This is bad JS practice